src/API/Logs/LoggerInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Logs;
+
+interface LoggerInterface
```

Comment:
Hello, welcome, and thanks for taking the time to review!
This interface relates of this part of the otel spec: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md#logger
I don't _think_ that there's anything in the spec that disallows your suggestion, and it could make it more useful... Let's see if anybody else has thoughts on this. (I'll be away for the next couple of days, so might not respond for a bit...)

Reply:
I think being compliant with PSR-3 would help a lot with adoption, I'm 👍🏼 on it.

Reply:
I'm happy with that, it's a very good suggestion. I will add psr-3 soon.

Reply:
added psr-3 compliance

Reply:
update: on review, the otel spec does explicitly disallow this: "The API is not intended to be called by application developers directly." so I've reverted psr-3.